src/bin/download_sysext.rs

```diff
@@ -291,7 +331,11 @@ struct Args {
 
     /// path to the Omaha XML file, or - to read from stdin
     #[argh(option, short = 'i')]
-    input_xml: String,
+    input_xml: Option<String>,
+
+    /// URL to fetch remote update payload
+    #[argh(option, short = 'u')]
+    payload_url: Option<String>,
```

Comment:
These two should be exclusive, only one mode is allowed

Reply:
I suppose that's not supported by argh out of the box, reading https://github.com/google/argh/issues/170.
So I had to manually check.